core/src/jmh/java/io/grpc/internal/SerializingExecutorBenchmark.java

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
I'm thinking probably the opposite. Why have microbenchmarks together? We don't do that for tests. We had 'benchmarks' because we had a big-fat benchmark, and it just so happened to be a convenient place for some other microbenchmarks. Do we actually gain anything by putting this in the benchmarks project?